src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,57 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
Replace `tolua_S:CheckParamSelf("cColor")` this check if the object is a instance of the class and will also print a warning if not.